operator/Makefile

```diff
@@ -65,11 +66,21 @@ run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 docker-build: test ## Build docker image with the manager.
-	docker build -t ${IMG} .
+	docker build -t ${IMG}:latest .
```

Comment:
why not use commit has initially?  We should not set latest, that will conflict with released images.

Reply:
it was set to latest by the IMG variable itself on ealier line